rtos/TARGET_CORTEX/mbed_rtx_conf.h

```diff
@@ -59,7 +59,7 @@
 #define OS_IDLE_THREAD_STACK_SIZE   (MBED_CONF_RTOS_IDLE_THREAD_STACK_SIZE + EXTRA_IDLE_STACK + EXTRA_IDLE_STACK_DEBUG)
 #endif
 
-#define OS_DYNAMIC_MEM_SIZE         0
+#define OS_DYNAMIC_MEM_SIZE        1024 
```

Comment:
This was added for PSOC6 testing but it should be reverted once the PSOC6 code no longer requires it. @vmedcy @morser499

Reply:
Removed in 
ec81b3e